corehq/apps/data_dictionary/util.py

```diff
@@ -417,3 +417,15 @@ def get_used_props_by_case_type(domain):
                 props_by_case_type[case_type_bucket.key] = []
             props_by_case_type[case_type_bucket.key].append(prop_bucket.key)
     return props_by_case_type
+
+
+def get_case_property_group_name_for_properties(domain, case_type_name):
+    case_group_name_for_property = {}
+    case_property_group_query = CaseProperty.objects.filter(
+        case_type__name=case_type_name, case_type__domain=domain
+    ).select_related('case_type').select_related('group').values_list('name', 'group__name')
```

Comment:
What do the `select_related` calls do here if we're only returning two columns?

Reply:
You are right. I didn't realize django would already handle that for the query and values_list.